pymecht/MatModel.py

```diff
@@ -1044,4 +1044,136 @@ def _energy_stress(self,A,B,mean_theta,sigma,aniso_fraction,**extra_args):
         tensors = np.array([np.outer(n,n) for n in vectors])
         energy = A*np.sum(self.Gamma_iterate*(np.exp(B*stretches)/B-1./B-stretches)*self._theta_weight_i)
         stress = A*np.sum(self.Gamma_iterate*(np.exp(B*stretches)-1)*self._theta_weight_i*tensors.T,axis=-1)
-        return energy,stress
\ No newline at end of file
+        return energy,stress
+
+class ARB(InvariantHyperelastic):
+    '''
+    Model of ARBitrary complexity for development purposes.
```

Comment:
The helper function should be updated to match other models.

Reply:
Will need to check if this is satisfactory.